create_payment_with_source.json

```diff
@@ -0,0 +1,9 @@
+{
+  "return_url":"https://example.com", 
+  "reference":"PP-7968", 
+  "amount": 100, 
+  "description": "PP-7968",
+  "internal": {
+    "source": "CARD_AGENT_INITIATED_MOTO"
+  }
+}
```

Comment:
What does this file do?

Reply:
ugh this is for personal use!! will delete

Reply:
deleted